src/Workleap.OpenApi.MSBuild/msbuild/tools/Workleap.OpenApi.MSBuild.targets

```diff
@@ -38,6 +38,10 @@
       <!-- The names of the Swagger documents to generate OpenAPI specifications for -->
       <!-- "v1" is the default Swagger document name. Users can specify multiple values separated by semicolons -->
       <OpenApiSwaggerDocumentNames Condition="'$(OpenApiSwaggerDocumentNames)' == ''">v1</OpenApiSwaggerDocumentNames>
+      
+      <!-- Use TreatWarningsAsErrors property unless it is opt-out with OpenApiIgnoreErrors -->
+      <OpenApiTreatWarningsAsErrors Condition="'$(OpenApiIgnoreErrors)' == 'true'">false</OpenApiTreatWarningsAsErrors>
+      <OpenApiTreatWarningsAsErrors Condition="'$(OpenApiIgnoreErrors)' != 'true'">$(TreatWarningsAsErrors)</OpenApiTreatWarningsAsErrors>
```

Comment:
I'm not sure where `OpenApiIgnoreErrors` comes from and why we would use it instead of `OpenApiTreatWarningsAsErrors` which actually fits with the existing .NET naming conventions, such as [CodeAnalysisTreatWarningsAsErrors](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/overview?tabs=net-8#treat-warnings-as-errors), which is a subset of `TreatWarningsAsErrors`

Also, assigning properties is usually done if not already UNSET. So this would be more appropriate with the standard and our other lines of code above:

```suggestion
      <!-- Use TreatWarningsAsErrors property unless it is opt-out with OpenApiTreatWarningsAsErrors, fallback to false if both are unset -->
      <OpenApiTreatWarningsAsErrors Condition="'$(OpenApiTreatWarningsAsErrors)' == ''">$(TreatWarningsAsErrors)</OpenApiTreatWarningsAsErrors>
      <OpenApiTreatWarningsAsErrors Condition="'$(OpenApiTreatWarningsAsErrors)' == ''">false</OpenApiTreatWarningsAsErrors>
```